modules/diskq/dqtool.c

```diff
@@ -225,7 +225,7 @@ dqtool_mode(int *argc, char **argv[])
   return NULL;
 }
 
-void
+void __attribute__((noreturn))
```

Comment:
I would avoid using `__attribute__` as much as it can be.
Possible it is either way better not to `exit(1)` in the `usage` method, it should just print the usage to `stderr` (do one thing).
The caller could do the `exit(1)`, but that is actually being the main could simple `return 1`.

(Possible this can be applied to the `pdbtool` as well.)